src/test/migrations/steps/metadata/dashboard/forestArea/index.ts

```diff
@@ -1,8 +1,9 @@
-import { Cycle, Unit } from 'meta/assessment'
+import { Cycle } from 'meta/assessment'
```

Comment:
files after this are reset to original state (everything under metadata/dashboard)